src/move_embargo_args.py

```diff
@@ -49,35 +48,32 @@ def parse_args():
     collections = namespace.collections
 
     # Dataset to move
-    # dataID must include
-    if not [x for x in (namespace.instrument, namespace.detector, namespace.band) if x is None]:
-        dataId = {'instrument': namespace.instrument, 'detector': namespace.detector,
-                  'band': namespace.band}
-    elif not [x for x in (namespace.instrument, namespace.detector, namespace.band, namespace.exposure) if x is None]:
-        dataId = {'instrument': namespace.instrument, 'detector': namespace.detector,
-                  'band': namespace.band, 'exposure': namespace.exposure}
-    else:
-        dataId = {'instrument': namespace.instrument}
+    dataId = {'instrument': namespace.instrument}
 
     # Define embargo period
-    embargo_period = astropy.time.TimeDelta(namespace.embargodays, format='jd')
-    now = astropy.time.Time.now()
-    int_now = int(now.datetime.strftime("%Y%m%d"))
+    embargo_period = astropy.time.TimeDelta(namespace.embargohours*3600., format='jd')
```

Comment:
I'm pretty sure you don't want 3600 JDs.
```suggestion
    embargo_period = astropy.time.TimeDelta(namespace.embargohours*3600., format='sec')
```